fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -1026,7 +1026,7 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
     @OptIn(DelicateCoroutinesApi::class)
     open fun downloadWallpapers() {
         GlobalScope.launch {
-            components.useCases.wallpaperUseCases.initialize()
+            components.wallpaperUseCases.initialize()
```

Comment:
There was a circular dependency issue adding usecases to the new AppStore middleware, so I had to move the wallpapers usecase up a level.